crates/holochain/src/core/workflow/app_validation_workflow/tests.rs

```diff
@@ -727,46 +727,50 @@ async fn test_private_entries_are_passed_to_validation_only_when_authored_with_f
         ..Default::default()
     };
 
-    let zomeset = InlineZomeSet::new_unique([("integrity", vec![entry_def], 0)], ["coordinator"])
```

Comment:
This is just formatting, right?

Reply:
Almost, yes. By adding a third argument to the `new_unique()` method, the arguments got wrapped onto new lines. So there has been added one more argument but other than that it's formatting.